assets/sass/patterns/molecules/site-header-nav-bar-primary.scss

```diff
@@ -80,13 +80,15 @@
 .nav-primary__item--first {
   display: flex;
   padding-top: 2px;
+
   a {
     background-image: url("../../assets/img/patterns/molecules/nav-primary-menu-ic.svg");
-    background-size: 22px;
-    background-repeat: no-repeat;
     background-position-x: 50px;
     background-position-y: -1px;
```

Comment:
Can we put the background position/repeat into one line please

```background: url("../../assets/img/patterns/molecules/nav-primary-menu-ic.svg") no-repeat 50px -1px;```

Reply:
I put in a line background position/repetition